docker/Readme.md

```diff
@@ -0,0 +1,86 @@
+# Docker for server
+
+## build
+The docker is easyly build using the following command - By default it will build the docker for the **develop** branch:
+
+```
+ docker build . -t iotagent-turtle 
+```
+
+If we want to build other branch, we could define the branch we want to use this way:
+```
+ docker build . -t iotagent-turtle --build-arg VERSION=integration-cb-httpserver
+```
+
+Everything will install in /proc/IotAgent-turtle inside the container. The default config file will be the following one:
+
+```
+{
+  "broker": "http://orion-ld:1026",
+  "logger": {
+    "path": "./logs/access.log",
+    "level": "debug",
+    "rotation": "20 days",
+    "retention": "1 months",
+    "format": "<level>{level: <8}</level> <green>{time:YYYY-MM-DD HH:mm:ss.SSS}</green> request id: {extra[request_id]} - <cyan>{name}</cyan>:<cyan>{function}</cyan> - <level>{message}</level>"
+  }
+}
+```
+
+## Use
+Let's suppose the image is named **io** and we want to expose port 5000 in order to work. We can also image that we want to remove the container when it finishes, we can do this way:
```

Comment:
Let's suppose the image is named **io** and we want to expose port 5000 in order to work. We can also imagine that we want to remove the container when it finishes, we can do it in the following way:

Reply:
ok